src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/Example1.java

```diff
@@ -9,11 +9,11 @@
 class Example1 {
   // xdoc section -- start
   abstract class AbstractFirst {}
-  abstract class Second {} // violation 'must match pattern '\^Abstract\.\+\$''
+  abstract class Second {} // violation 'must match pattern'
```

Comment:
I believe the example is understandable without mentioning the pattern.

Reply:
I like without pattern as pattern visible in config.